consul/base.py

```diff
@@ -1409,17 +1474,47 @@ def node(self,
                 '/v1/catalog/node/%s' % node,
                 params=params)
 
+        def _service(self,
+                     internal_uri,
+                     index=None,
+                     wait=None,
+                     tag=None,
+                     consistency=None,
+                     dc=None,
+                     near=None,
+                     token=None,
+                     node_meta=None):
+            params = []
+            dc = dc or self.agent.dc
+            if dc:
+                params.append(('dc', dc))
+            if tag:
+                params.append(('tag', tag))
+            if index:
+                params.append(('index', index))
+                if wait:
+                    params.append(('wait', wait))
+            if near:
+                params.append(('near', near))
+            token = token or self.agent.token
+            if token:
+                params.append(('token', token))
+            consistency = consistency or self.agent.consistency
+            if consistency in ('consistent', 'stale'):
+                params.append((consistency, '1'))
+            if node_meta:
+                for nodemeta_name, nodemeta_value in node_meta.items():
+                    params.append(('node-meta', '{0}:{1}'.
```

Comment:
Why not use f-string?

Reply:
consistency with the rest of the codebase.

Reply:
ok